truss/tests/test_model_inference.py

```diff
@@ -81,6 +82,62 @@ def test_binary_request(sklearn_rfc_model):
         assert np.shape(predictions["probabilities"]) == (1, 3)
 
 
+@pytest.mark.integration
+def test_model_load_failure_truss():
+    with ensure_kill_all():
+        truss_root = Path(__file__).parent.parent.parent.resolve() / "truss"
+        truss_dir = truss_root / "test_data" / "model_load_failure_test"
+        tr = TrussHandle(truss_dir)
+
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=False)
+
+        # Sleep a few seconds to get the server some time to  wake up
+        time.sleep(10)
+
+        truss_server_addr = "http://localhost:8090"
+
+        def handle_request_exception(func):
+            def wrapper(*args, **kwargs):
+                try:
+                    return func(*args, **kwargs)
+                except RequestException:
+                    print("Exception")
+                    return False
+
+            return wrapper
+
+        @handle_request_exception
+        def _test_liveness_probe(expected_code):
```

Comment:
For a followup PR: We should refactor these integration test utilities as we're starting to use them in more tests.